cloudformation/services/concordia-service/service.yaml

```diff
@@ -39,31 +39,34 @@ Parameters:
 
 Mappings:
     ClusterMapping:
+        RedisEndpointMap:
+            dev: 'crowd-dev-redis.hfkoms.0001.use1.cache.amazonaws.com'
```

Comment:
Does the stack which creates these resources export their outputs? If there's a reason why we can't do that, I'm thinking the answer might be migrating to Route53 so we could say "redis.{environment}.internaltld"